policy_wizard/views.py

```diff
@@ -62,14 +64,16 @@ def policy_templates_list_view(request):
     if role=='Instructor' or role=='Administrator':
 
         if role=='Instructor':
-            try: #if a policy has already been published for this course ...
-                #Get the published policy
-                publishedPolicy = Policies.objects.get(context_id=request.session['context_id'])
-                #Render the published policy
+            try: #If there is an active published policy for this course, get it. (Only 1 active published policy expected.)
+                publishedPolicy = Policies.objects.get(context_id=request.session['context_id'], is_active=True)
+                # Render the active published policy
                 return render(request, 'instructor_published_policy.html', {'publishedPolicy': publishedPolicy})
-            except Policies.DoesNotExist: #If no such policy exists ...
+            except Policies.MultipleObjectsReturned: #If multiple active published policies exist (which should never happen)...
+                return HttpResponse("Something went wrong #@$%. Contact the site admin at atg@fas.harvard.edu.")
```

Comment:
This might be a good place to return an HTTP 500 code to indicate an unexpected condition on the server. The [HttpResponseServerError](https://docs.djangoproject.com/en/2.0/ref/request-response/#django.http.HttpResponseServerError) class could be substituted for HttpResponse to do that (or you can pass in the status manually). Also, I'm wondering if it might be worth factoring the ATG support email address into a secure setting, to keep things somewhat generic.

Reply:
Just implemented that in d318998. Thanks for the feedback.